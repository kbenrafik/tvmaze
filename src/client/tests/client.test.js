import {
  makeClient
} from '../index';

describe('makeClient', () => {
  it('should make a client', () => {
    const client = makeClient();
    expect(client).toMatchSnapshot();
  })
});
