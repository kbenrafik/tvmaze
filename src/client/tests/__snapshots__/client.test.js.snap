// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`makeClient should make a client 1`] = `
Client {
  "episode": Episode {
    "axios": [Function],
    "host": "http://api.tvmaze.com/",
  },
  "host": "http://api.tvmaze.com/",
  "season": Season {
    "axios": [Function],
    "host": "http://api.tvmaze.com/",
  },
  "show": Show {
    "axios": [Function],
    "host": "http://api.tvmaze.com/",
  },
}
`;
